src/app/cake/cake.model.ts

```diff
@@ -0,0 +1,10 @@
+export type Status = 'Available' | 'LastUnits' | 'OutOfStock';
+
+export interface CakeInterface {
```

Comment:
As a good practice, you can add the letter I before the name of your interfaces, to identify them versus the classes. Ex. `IBread` or `iBread`.